controller/projects.go

```diff
@@ -117,6 +161,26 @@ func (p *Projects) Get(id uuid.UUID) (*model.Project, error) {
 	return &proj, nil
 }
 
+func (p *Projects) GetProjectListForUser(userID uuid.UUID, auth *auth.Authenticator, ctx context.Context) (*model.ProjectList, error) {
```

Comment:
I feel in the future we should move this to the user resource, so you retrieve user and all related entities like projects, tutorial completions etc. Just as a note.

Reply:
Yeah sounds good!